src/app/config.py

```diff
@@ -12,6 +14,7 @@
 class Config:
     TESTING = False
     DEBUG = False
+    SECRET_KEY = "07c56596b48f48b3a749a6969e64fadc"
```

Comment:
I don't believe this belongs in the repository. To generate a value by default it might make sense to do something like

```python
...

if not (SECRET_KEY := os.environ.get("SECRET_KEY")):
    import secrets
    SECRET_KEY = secrets.token_hex()
    with open('.env', 'a') as f:
        f.write(f"SECRET_KEY={SECRET_KEY}")


...
```